support/rally.js

```diff
@@ -23,19 +23,32 @@ class Rally {
    *          "y":"x_FEzRu9m36HLN_tue659LNpXW6pCyStikYjKIWI5a0",
    *          "kid":"Public key used in JWS spec Appendix A.3 example"
    *        }
+   * @param {Boolean} enableDevMode
+   *        Whether or not to initialize Rally.js in developer mode.
+   *        In this mode we ignore problems when communicating with
+   *        core add-on.
    */
-  async initialize(keyId, key) {
+  async initialize(keyId, key, enableDevMode) {
     console.debug("Rally.initialize");
 
     this._validateEncryptionKey(keyId, key);
 
     this._keyId = keyId;
     this._key = key;
+    this._enableDevMode = !!enableDevMode;
```

Comment:
nit, but I've had experienced JS devs see this method of casting/coercing a boolean value and not known wtf I was doing :)

Anyway up to you but I'd consider:

```suggestion
    this._enableDevMode = Boolean(enableDevMode);
```

Just because it's pretty crystal-clear what's happening.